rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -373,9 +374,15 @@ static NEVER_INLINE error_code savedata_op(ppu_thread& ppu, u32 operation, u32 v
 
 		while (funcList)
 		{
+			// Yield
+			lv2_obj::sleep(ppu);
```

Comment:
:man_facepalming: Thanks

Reply:
can you fix gow 3 pls ? ty about freezes